pkgs/by-name/zs/zs/package.nix

```
@@ -0,0 +1,45 @@
+{ lib, fetchFromGitea, buildGoModule, installShellFiles }:
+
+buildGoModule rec {
+  pname = "zs";
+  version = "0.4.1";
+
+  src = fetchFromGitea {
+    domain = "git.mills.io";
+    owner = "prologic";
+    repo = "zs";
+    rev = version;
+    hash = "sha256-V8+p19kvVh64yCreNVp4RVdkJkjrq8Q5VbjaJWekZHY=";
+  };
+
+  vendorHash = "sha256-KXcYTYO4wnWOup5uJ6T+XwthX5l2FL02JyOt1Nv51Sg=";
+
+  ldflags = [
+    "-w"
+    "-X=main.Version=${version}"
+    "-X=main.Commit=${src.rev}"
+    "-X=main.Build=1970-01-01T00:00:00+00:00"
+  ];
+
+  nativeBuildInputs = [installShellFiles];
```

```suggestion
  nativeBuildInputs = [ installShellFiles ];
```
Small nit. See [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#code-conventions) for code conventions.